src/test/java/org/sagebionetworks/bridge/reporter/worker/SignUpsReportGeneratorTest.java

```diff
@@ -0,0 +1,67 @@
+package org.sagebionetworks.bridge.reporter.worker;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import org.sagebionetworks.bridge.reporter.helper.BridgeHelper;
+import org.sagebionetworks.bridge.reporter.request.ReportType;
+import org.sagebionetworks.bridge.rest.model.AccountStatus;
+import org.sagebionetworks.bridge.rest.model.SharingScope;
+import org.sagebionetworks.bridge.rest.model.Study;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+
+import com.google.common.collect.Lists;
+
+public class SignUpsReportGeneratorTest {
```

Comment:
I also realized that in TestNG, the order is actual and then expected, the opposite of JUnit. So I am going to reverse all the tests I've added.